docs/supporting/requirements.md

```diff
@@ -19,16 +19,16 @@ We recommend:
 
 ## Platforms
 
-VS Code has been tested on the following platforms:
+VS Code is supported on the following platforms:
 
 * OS X El Capitan (10.11+)
 * Windows 8.0, 8.1 and 10, 11 (32-bit and 64-bit)
-* Linux (Debian): Ubuntu Desktop 16.04, Debian 9
-* Linux (Red Hat): Red Hat Enterprise Linux 7, CentOS 8, Fedora 24
+* Linux (Debian): Ubuntu Desktop 14.04, Debian 9
```

Comment:
`Ubuntu 14.04` is end of life in 2022 https://ubuntu.com/blog/ubuntu-14-04-and-16-04-lifecycle-extended-to-ten-years

Reply:
According to the article, it seems they're extending that EOL to 2024. Pretty sure Fedora 34 reaches EOL this year, though.